critiquebrainz/data/dump_manager.py

```diff
@@ -314,11 +316,12 @@ def create_base_archive(*, location, meta_files_dir=None):
         return " + %s/cbdump.tar.bz2" % location
 
 
-def create_reviews_archive(*, location, meta_files_dir=None, license_id=None):
+def create_reviews_archive(connection, *, location, meta_files_dir=None, license_id=None):
     """Creates a dump of reviews filtered on the given license_id, their revisions and
        the avg. rating tables.
 
     Args:
+        connection (object): for supporting execution within a transaction
```

Comment:
everything is an object in python so the type hint here is not very useful, you should be more specific. :)

Reply:
@paramsingh something like `object returned by db.engine.begin()` ?